src/scene/sky.js

```diff
@@ -66,9 +66,11 @@ class Sky {
                     toneMapping: (pass === SHADER_FORWARDHDR ? TONEMAP_LINEAR : scene.toneMapping)
                 });
             }
+
+            return this.shader;
```

Comment:
shouldn't be using `this.shader` anymore?

Reply:
I change this one as well .. but for Material instance, the shader is / should be set. But for consistency, it should be done from outside the call, not inside.

But I'll be adding assignment to internal variations shader there I think soon, as that will be needed for my next steps.